src/composables/useTexture/index.ts

```diff
@@ -114,9 +114,10 @@ export async function useTexture<TextureMap extends PBRUseTextureMap>(
 
 export async function useTexture(
   paths: readonly [string] | string[] | PBRUseTextureMap,
+  manager?: LoadingManager,
```

Comment:
@hawk86104 the default value doesn't need to be instantiated like manager: `new LoadingManager()` ?

Reply:
https://threejs.org/docs/index.html#api/en/loaders/TextureLoader

Reply:
The three.js documentation already states that default is [THREE.DefaultLoadingManager](https://threejs.org/docs/index.html#api/en/loaders/managers/LoadingManager)

Reply:
I have also tested it in my own project, and there were no issues.